snapcraft/internal/errors.py

```diff
@@ -517,3 +517,17 @@ class InvalidExtractorValueError(MetadataExtractionError):
 
     def __init__(self, path: str, extractor_name: str) -> None:
         super().__init__(path=path, extractor_name=extractor_name)
+
+
+class ScriptletError(SnapcraftError):
+    pass
```

Comment:
I've noticed that instead of pass we could also add a simple docstring to this explaining that it is a base class for others (and even if it will bring in inconcistency, we should add Base to the name, wdyt?)

Reply:
Agreed on both points, done.